src/ORM/Table.php

```diff
@@ -1601,6 +1601,123 @@ public function deleteAll($conditions)
     }
 
     /**
+     * Deletes records individually by reading batches that match the conditions. For each
+     * matched record Table::delete() is executed providing support for behaviors and events.
+     *
+     * This method will continue execution until all records are deleted, or one of the events is stopped.
+     * If a record fails to be deleted the operation will stop (optional).
+     *
+     * This method will *not* trigger afterDeleteCommit events. The event afterDeleteEachCommit will be
+     * broadcast after each batch is committed.
+     *
+     * This method will *not* accept query options that alter the size of a batch, such as limit, offset
+     * and page.
+     *
+     * ### Options
+     *
+     * - 'finder' string Defaults to 'all'. The custom finder to use when reading batches.
+     * - 'options' array Defaults to []. The options to pass to the finder when reading batches.
+     * - `atomic` boolean Defaults to true. When true the deletion happens within a transaction.
+     * - 'stopOnFailure' boolean Defaults to true. Stop deleting records when a record fails to be deleted.
+     * - 'batch' integer Defaults to 1. Number of records to read in batches, zero reads all records.
+     * - `checkRules` boolean Defaults to true. Check deletion rules before deleting the record.
```

Comment:
You have a mix of backticks and quotes here. Could they all be made into backticks? Those generate nicer API docs.